### ISSUE TITLE
bug: Dockerfile rule apt_get_missing_yes_flag not handling the "-v" after the package

### ISSUE THREAD

**Issue by tellierd** [aquasecurity/defsec#1173]
**Describe the bug**

The rules [ds021](https://github.com/aquasecurity/defsec/blob/master/internal/rules/policies/docker/policies/apt_get_missing_yes_flag_to_avoid_manual_input.rego) throws this error ```HIGH: '-y' flag is missed``` when the "-y" is after the package name ```apt-get install apt-utils -y```

Referring to this [file](https://github.com/aquasecurity/defsec/blob/master/internal/rules/policies/docker/policies/apt_get_missing_yes_flag_to_avoid_manual_input_test.rego), I also noticed, this is not part of the tests. I think it could be nice to add it too.

**To Reproduce**

1. Create a Dockerfile
2. Add ```RUN apt-get install apt-utils -y``` and save it
3. In the same folder as the Dockerfile, run this Trivy (0.37.1) command: ```trivy config .```

**Expected behavior**

Should pass without error

**Output of your trivy command with --debug flag**

```
$trivy config --debug .

DEBUG   Severities: ["UNKNOWN" "LOW" "MEDIUM" "HIGH" "CRITICAL"]
DEBUG   cache dir:  /Users/username/Library/Caches/trivy
INFO    Misconfiguration scanning is enabled
DEBUG   Policies successfully loaded from disk
DEBUG   Walk the file tree rooted at '.' in parallel
DEBUG   OS is not detected.
INFO    Detected config files: 1
DEBUG   Scanned config file: Dockerfile

Dockerfile (dockerfile)

Tests: 24 (SUCCESSES: 22, FAILURES: 2, EXCEPTIONS: 0)
Failures: 2 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

HIGH: '-y' flag is missed: 'apt-get install apt-utils -y'
═══════════════════════════════════════════════════════════════
'apt-get' calls should use the flag '-y' to avoid manual user input.

See https://avd.aquasec.com/misconfig/ds021
─────────────────────────────────────────────
 Dockerfile:2
─────────────────────────────────────────────
   2 [ RUN apt-get install apt-utils -y
─────────────────────────────────────────────
```

**Additional context**

I got this when running with Trivy version 0.37.1 on updated rules db
